Components/badge/badgeOnAvatar/styles.css

```diff
@@ -0,0 +1,69 @@
+@import url(/rootStyles.css);
+@import url(../../avatar/styles.css);
+
+.componentAvatarBadge {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.group {
+  display: flex;
+  justify-content: space-around;
+}
+.avatarBadgeContainer {
+  position: relative;
+}
+.badgeOnline {
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  background-color: green;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+.badgeOffline {
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  background-color: red;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+.badge-S{
+  width: 0.7rem;
+  height: 0.7rem;
+}
+.badge-XS{
+  width: 0.5rem;
+  height: 0.5rem;
+}
+.badge-L{
+  width: 1.5rem;
+  height: 1.5rem;
+}
+.badge-RL{
+  width: 1.5rem;
+  height: 1.5rem;
+  bottom:3px;
+  right:5px;
+}
+.badge-RXS{
```

Comment:
Again, for positions, utility classes could be made here for customization. Cause everything looks same except the position.